x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/constants.ts

```diff
@@ -22,13 +21,10 @@ export const SEARCHABLE_FIELDS: Readonly<string[]> = [
   `entries.value`,
 ];
 
-export const HOST_ISOLATION_EXCEPTIONS_LIST_TYPE: ExceptionListType =
-  ExceptionListTypeEnum.ENDPOINT_HOST_ISOLATION_EXCEPTIONS;
-
 export const HOST_ISOLATION_EXCEPTIONS_LIST_DEFINITION: CreateExceptionListSchema = {
   name: ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_NAME,
   namespace_type: 'agnostic',
   description: ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_DESCRIPTION,
   list_id: ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID,
-  type: HOST_ISOLATION_EXCEPTIONS_LIST_TYPE,
+  type: ExceptionListTypeEnum.ENDPOINT_HOST_ISOLATION_EXCEPTIONS,
```

Comment:
This is nice! Although, I notice for some reason the trusted_apps list type is called `endpoint`😱 . We should update it to match the other names. Perhaps `endpoint_trusted_apps`. I'll do this in my PR if you agree.

Reply:
Yeah, you can do that as long as it does not break anything. This custom `type` value was introduced at some point after TA and we never got to update TA to also have its own type.